src/qt/bitcoin.cpp

```diff
@@ -60,9 +60,24 @@ Q_IMPORT_PLUGIN(QXcbIntegrationPlugin);
 Q_IMPORT_PLUGIN(QWindowsIntegrationPlugin);
 #elif defined(QT_QPA_PLATFORM_COCOA)
 Q_IMPORT_PLUGIN(QCocoaIntegrationPlugin);
+#elif defined(QT_QPA_PLATFORM_ANDROID)
+Q_IMPORT_PLUGIN(QAndroidPlatformIntegrationPlugin);
 #endif
 #endif
 
+#if defined (Q_OS_ANDROID)
+#include <QtAndroidExtras/QtAndroid>
+const QVector<QString> permissions({"android.permission.ACCESS_COARSE_LOCATION",
```

Comment:
Why should Bitcoin Core ever request the user's location?

The permission list looks like a security/privacy nightmare TBH :)

Reply:
The permissions list may require review to ensure it is safe. This whole process raises concerns, and questions as to whether permissions are a one time thing or whether they should be requested everytime, as well as management if the process in the background. Due to costs of mobile data, some users may want to opt for wi-fi only or only allowing network access when they are actually using the app. Permissions will be headache IMO but no way around it for newer android versions.